jaas/src/main/resources/jagger.properties

```diff
@@ -0,0 +1 @@
+chassis.master.reporting.root.path=classpath:
```

Comment:
why is it called 'jagger.properties'?
please, add comment: what is it? why is it required? when it should be changed?